src/posts/2024-06-21-cilium/index.md

```diff
@@ -0,0 +1,9 @@
+---
+date: '2024-06-24T17:20:05.000Z'
+title: 'Enabling Multicast securely with IPsec in the cloud native landscape with Cilium'
```

Comment:
```suggestion
title: 'Cilium: Open-source eBPF-based networking, security, observability'
```

Reply:
I am missing this too frequently, thanks!